src/NzbDrone.Core/IndexerSearch/AlbumSearchCommand.cs

```diff
@@ -0,0 +1,11 @@
+﻿using NzbDrone.Core.Messaging.Commands;
+
+namespace NzbDrone.Core.IndexerSearch
+{
+    class AlbumSearchCommand : Command
+    {
+        public int AlbumId { get; set; }
```

Comment:
Should this be changed to accept a list of Ids, similar to the episode search command?